templates/glanceapi/config/ceph.conf

```diff
@@ -1,3 +1,6 @@
 [global]
 fsid = {{ .CephClusterFSID }}
 mon_host = {{ .CephClusterMonHosts }}
+
+[client.{{ .CephUser }}]
+keyring = /etc/ceph/ceph.client.openstack.keyring
```

Comment:
s/openstack/{{ .CephUser }}

Reply:
If I s/openstack/{{ .CephUser }} then I'll need to make the filename variable too as it's hard coded as:

  templates/glanceapi/config/ceph.client.openstack.keyring

E.g. If I do change it I'd need to update `templates/glanceapi/config/glance-api-config.json` too. 

My test of this with our external cluster which uses different pool and cephx user names succeeded but required lines 5 and 6. That's how glance figures out which keyring to use. As per the glance doc "A keyring must be set for this user in the Ceph configuration file, e.g. with a user" 

https://docs.openstack.org/glance/latest/configuration/configuring.html#configuring-the-rbd-storage-backend

Reply:
@fultonj I did it here [1], and the src file is renamed as expected and accessible from the container. But if not required here, I'm fine with this.

[1] https://github.com/openstack-k8s-operators/cinder-operator/blob/master/templates/cinder/config/cinder-volume-config.json#L30

Reply:
@fmount OK, thanks for the suggestion. I've updated accordingly. 

https://github.com/openstack-k8s-operators/glance-operator/pull/32/files#diff-a68e93cd8bb32386db24d673a79424702323b2173163359eead0c2c72cc48bf6

Reply:
+1 thanks John!